mlir/lib/Dialect/Vector/IR/VectorOps.cpp

```diff
@@ -6212,6 +6214,7 @@ bool WarpExecuteOnLane0Op::areTypesCompatible(Type lhs, Type rhs) {
 
 Value mlir::vector::makeArithReduction(OpBuilder &b, Location loc,
                                        CombiningKind kind, Value v1, Value acc,
+                                       arith::FastMathFlagsAttr fastmath,
```

Comment:
Isn't this unrelated to this change?

Reply:
I need this to propagate fastmath attributes. I figured this is a relatively small change, but I can split it into a separate change if this makes code review difficult -- let me know.

Reply:
The noise level is low, so that's not a concern. However, these attributes might affect things in a very subtle way and that's why I feel the deserve a dedicated commit. I would happily review that and accept (it's very straightforward).

Reply:
Sure, here it is https://github.com/llvm/llvm-project/pull/75846